test/e2e/upgrades/ingress.go

```diff
@@ -176,6 +176,15 @@ func (t *IngressUpgradeTest) verify(f *framework.Framework, done <-chan struct{}
 	By("comparing GCP resources post-upgrade")
 	postUpgradeResourceStore := &GCPResourceStore{}
 	t.populateGCPResourceStore(postUpgradeResourceStore)
+
+	// Ignore certain fields in compute.Firewall that we know will change
+	// due to the upgrade/downgrade.
+	// TODO(rramkumar): Remove this once glbc 0.9.8 is released.
+	t.resourceStore.Fw.Allowed = nil
```

Comment:
Or could we just do `Fw.Allowed.Ports=nil`?

Reply:
Done.

Reply:
Sorry, seems like `Allowed` is a slice of `FirewallAllowed` and it would be a bit messy to tweak that. I think what you have before is good enough...

Reply:
Yep, just realized that. Changed it back.